src/sdk/conference/subscription.js

```diff
@@ -298,6 +299,18 @@ export class Subscription extends EventDispatcher {
       writable: true,
       value: stream,
     });
+    /**
+     * @member {Owt.Base.TransportSettings} transport
+     * @instance
+     * @readonly
+     * @desc Transport settings for the publication.
```

Comment:
for the subscription. Same for the line below this one.

Reply:
Fixed.